bundles/FileExplorerBundle/public/js/startup.js

```diff
@@ -0,0 +1,73 @@
+pimcore.registerNS("pimcore.bundle.file_explorer.startup");
+
+pimcore.bundle.file_explorer.startup = Class.create({
+
+    initialize: function () {
+        document.addEventListener(pimcore.events.preMenuBuild, this.preMenuBuild.bind(this));
+    },
+
+    preMenuBuild: function (event) {
+        const menu = event.detail.menu;
+        this.user = pimcore.globalmanager.get('user');
+        this.toolbar = pimcore.globalmanager.get('layout_toolbar');
+        const systemInfoMenuItems = this.getSystemInfoMenu();
+
+        const filteredMenu = menu.extras.items.filter(function (item) {
+            return item.itemId === 'pimcore_menu_extras_system_info';
+        });
+
+        if (filteredMenu.length > 0) {
```

Comment:
No need to check and create main menu anymore as the with new approach the menu is always there. please see https://github.com/pimcore/pimcore/pull/13887/commits/a7a6083f190e384fa0cd1b1a97c85d25b5fb6c09#diff-594df434516f523fb7936f5c0652aaa376b5b4e4a10f410e423aa7926a1cdad6

Reply:
We still have to do it because System Settings & Tools is now it's own bundle, so if it is disabled, we still need to add the menu item.

Reply:
Ah yes sorry you are right :) 
Removed it.